app/manager-frontend/src/pages/ActivitySource/list.jsx

```diff
@@ -0,0 +1,151 @@
+import {
+  Heading,
+  VStack,
+  Button,
+  useDisclosure,
```

Comment:
is this the only file in the `pages/ActivitySource/` directory?

if so, it doesn't need to be in a folder, it can just be `ActivitySourceList.jsx` in `pages/`.

If not, what else is there?

Ideally we shouldn't have multiple pages in folders. Most pages should just be one file and be `pages/<PageName>.jsx`

However, if there are components or other files used **only by that page**, then use the format `pages/<PageName>/index.jsx`